http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/api/TestLeoRoutes.scala

```diff
@@ -31,6 +37,14 @@ trait TestLeoRoutes { this: ScalatestRouteTest with Matchers with CommonTestData
   implicit private val nr = FakeNewRelicMetricsInterpreter
   implicit def unsafeLogger = Slf4jLogger.getLogger[IO]
 
+  val mockGoogleDirectoryDAO = new MockGoogleDirectoryDAO()
+  Await.result(
```

Comment:
Btw, I try to prefer `future.futureValue` over `Await.result(future, duration)` in unit tests. `futureValue` comes from ScalaTest and has some improvements around polling, timeouts, error messages, etc.